### PR TITLE
Cirrus: Add/fix documentation links

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ### Latest Version: 0.12.1
 ### Status: Active Development
 
-### Continuous Integration: [![Build Status](https://api.cirrus-ci.com/github/containers/libpod.svg)](https://cirrus-ci.com/github/containers/libpod)
+### Continuous Integration [![Build Status](https://api.cirrus-ci.com/github/containers/libpod.svg)](https://cirrus-ci.com/github/containers/libpod/master) [CI documentation](contrib/cirrus/README.md)
```
Comment (mheon) [inline README.md]: Can you throw the Docs link on a new line?

Comment (cevich) [reply README.md]: certainly.

**Comment by mheon** [comment]
LGTM
@baude @rhatdan @TomSweeneyRedHat PTAL

**Comment by cevich** [comment]
@mheon here's what it looks like rendered: https://github.com/containers/libpod/blob/e209c9ec7d94f8695fa931a249f6026545cc2360/README.md

TBH, I still think looks a bit strange, something is "off" about the flow...I'll tinker some more...

**Comment by cevich** [comment]
@baude @rhatdan @TomSweeneyRedHat PTAL [latest incarnation rendered](https://github.com/containers/libpod/blob/d038497b09ac064fafe4f921c0fc43b828a707a3/README.md)  I re-arranged the sections a bit and updated some wording.  I think it flows better, whatchall think?

**Pull request merged**